### PR TITLE
Dashboard Cerl_url

### PR DESCRIPTION
**Pull request by ElakkiyaMohanRajan**
#### What does this PR do? Please provide some context
  - change certificate URL if course organization is in SHARED_ORG.
  - SHARED_ORG  list could be [Shared , All ,etc..]
#### Where should the reviewer start?
-  DashBoard --> View Certificate --> URL redirects to default site domain.
#### How c - BEFORE : Certificate URL redirects to default domain.
     AFTER : For Employee site certificate URL will redirect based on course org .if it is in 
    SHARED_ORG it will redirect to CERT_URL domain

#### What are the relevant TFS items? (list id numbers)
Bug:102033
#### Definition of done:
- [ ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [ ] For large or complex change: schedule an in-person review session
- [ ] This change has appropriate test coverage

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging from a feature branch into the development branch then "flatten" or "squash" commits
1. If merging from the development branch into master (or porting changes from upstream) then use github's UI to get review feedback, but use the git command line interface to complete the actual merge.

#### Reminders AFTER merging
1. Delete the remote feature branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged from the development branch into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc). In this case, use github's UI for feedback and the git command line interface for the actual merge.

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -384,9 +385,16 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
         # showing the certificate web view button if certificate is ready state and feature flags are enabled.
         if has_html_certificates_enabled(course_overview.id, course_overview):
             if course_overview.has_any_active_web_certificate:
+                config_value = configuration_helpers.get_configuration_value('SHARED_ORG', '')
+                course_obj = CourseSerializer()
+                course_org = course_obj.get_org(course_overview)
+                if course_org in config_value:
+                    cert_url = configuration_helpers.get_configuration_value('CERT_URL', '')
```
Comment (sdolenc) [inline common/djangoapps/student/views.py]: out of curiosity: what's an example value for cert_url ?

File: common/djangoapps/student/views.py [hunk 2]
```diff
@@ -384,9 +385,16 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
...
+                else:
+                    cert_url = ''
                 status_dict.update({
                     'show_cert_web_view': True,
-                    'cert_web_view_url': get_certificate_url(course_id=course_overview.id, uuid=cert_status['uuid'])
```
Comment (sdolenc) [inline common/djangoapps/student/views.py]: out of curiosity: what's an example response for get_certificate_url ?

File: common/djangoapps/student/views.py [hunk 3]
```diff
@@ -384,9 +385,16 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
...
+                    'cert_web_view_url': cert_url + get_certificate_url(course_id=course_overview.id, uuid=cert_status['uuid'])
```
Comment (sdolenc) [inline common/djangoapps/student/views.py]: concatenating fully qualified urls with protocols isn't good practice for multiple reasons. that's why i'm curious what example outputs are. A more standard approach is to encode the second url and append it as a query parameter of the first url. the handler for the first url then needs to be able to parse and utilize the second url (maybe do some work before redirecting to the second url, etc).

File: common/djangoapps/student/views.py (hunk 1 above)
Comment (ElakkiyaMohanRajan) [reply common/djangoapps/student/views.py]: The cert_url will be the url which we are setting up in Site_Configuration. If the organization is Shared as per the requirement we are redirecting the site to Cert_url .

**Pull request merged**